docs/en/compute/Presto-Iceberg.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: global
+title: Querying Iceberg Tables with Presto and Alluxio
+nickname: Presto-Iceberg
+group: Compute Integrations
+priority: 2
+---
+
+Presto has introduced support for [Iceberg tables](https://iceberg.apache.org/) in version 0.256.
+
+This document describes how to use Presto to query Iceberg tables through Alluxio.
+
+* Table of Contents
+{:toc}
+
+In order to use Presto to query an Iceberg table, make sure you have set up Presto, Hive metastore 
+and Alluxio. If you haven't done so, please refer to the 
+[guide]({{ '/en/compute/Presto.html' | relativize_url }}) on general Presto installation 
+and configuration. Most of that guide apply for Iceberg workflows as well, and this document 
+covers the specific instructions for working with Iceberg tables.
+
+Before proceeding, make sure you have a working setup of Presto, Hive metastore and Alluxio, 
+and Presto can access data through Alluxio's filesystem interface. 
+The [guide]({{ '/en/compute/Presto.html' | relativize_url }}) mentions two approaches that 
+Presto can access data through Alluxio: via the filesystem interface, and via the catalog service. 
+Alluxio has not added support for Iceberg in the catalog service yet, so for Iceberg tables Presto 
+needs to access them through Alluxio's filesystem interface.
+
+## Install Alluxio client jar to Presto Iceberg connector
+
+Copy the Alluxio client jar located at `{{site.ALLUXIO_CLIENT_JAR_PATH}}` into Presto Iceberg 
+connector's directory located at `${PRESTO_HOME}/plugin/iceberg/`. Then restart the Presto server:
+
+```console
+$ ${PRESTO_HOME}/bin/launcher restart
+```
+
+Also note that the same client jar file needs to be copied to Hive's `lib` directory.
+In case you haven't done so, please refer to the 
+[section]({{ '/en/compute/Hive.html' | relativize_url }}#basic-setup) on setting up Hive to 
+work with Alluxio.
+
+## Configure Presto to use the Iceberg connector
+
+Presto reads and writes an Iceberg table using the 
+[Iceberg connector](https://prestodb.io/docs/current/connector/iceberg.html). To enable the Iceberg 
+connector, create a catalog for Iceberg connector in Presto's installation directory as 
+`${PRESTO_HOME}/etc/catalog/iceberg.properties`:
+
+```properties
+connector.name=iceberg
+hive.metastore.uri=thrift://localhost:9083
+```
+
+Change the Hive connection URI to match your Hive setup.
+
+## Create a schema and an Iceberg table
+
+For demonstration purposes, we will create an example schema and an Iceberg table.
+
+Launch the Presto CLI client, and run the following statements:
+
+```sql
+CREATE SCHEMA iceberg_test;
```

Comment:
nit: should we do 'use iceberg.default' first?  or `CREATE SCHEMA iceberg.iceberg_test;` ?

Reply:
I accidentally left out the command to start the client with `--catalog iceberg`. This has been fixed in the last commit.

Reply:
Cool, using --catalog is great